crypto/ed25519/ed25519.go

```diff
@@ -15,34 +14,24 @@ import (
 
 //-------------------------------------
 
-var _ crypto.PrivKey = PrivKeyEd25519{}
+var _ crypto.PrivKey = PrivKey{}
 
 const (
 	PrivKeyAminoName = "tendermint/PrivKeyEd25519"
 	PubKeyAminoName  = "tendermint/PubKeyEd25519"
+	// PubKeySize is the number of bytes in an Ed25519 signature.
+	PubKeySize = 32
 	// Size of an Edwards25519 signature. Namely the size of a compressed
 	// Edwards25519 point, and a field element. Both of which are 32 bytes.
-	SignatureSize = 64
+	PrivateKeySize = 64
```

Comment:
Err somethings wrong with the comments here.

Pubkey size = 32 bytes is currently listed, but the comment says its the signature size.

Similar the PrivateKeySize's comment is for SignatureSize as well